packages/authentication/src/index.ts

```diff
@@ -2,5 +2,7 @@
 // Node module: @loopback/authentication
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
+// NOTE(bajtos) This file is used by VSCode/TypeScriptServer at dev time only
```

Comment:
This comment applies to `packages/authentication/index.ts` only, please revert it here.